packages/build/bin/build-css.js

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/env node
-
 const chalk = require('chalk')
 const program = require('commander')
 const path = require('path')
 
 const { css } = require('../index')
-const defaultInputFilePath = path.join(process.cwd(), 'dist', 'css', 'index.js')
-const defaultOutputDirectoryPath = path.join(process.cwd(), 'dist', 'css')
+const defaultInputFilePath = path.join('dist', 'css', 'index.js')
```

Comment:
I put the cross-plat separators back in, kept cwd out, tested and it worked.

Reply:
Cool